dockerfiles/build.sh

```diff
@@ -43,7 +43,7 @@ build_directory() {
 
 build_all() {
   # loop on all directories and call build.sh script if present
-  for directory in */ ; do
+  for directory in 'theia-dev/' 'theia/' 'theia-endpoint-runtime-binary/' 'theia-vsix-installer/' ; do
```

Comment:
As you're hardcoding path, could you use instead:

```suggestion
  for directory in $(ls -d */ | sort) ; do
```